src/NodaTime.Web/Providers/GoogleCredentialProvider.cs

```diff
@@ -16,14 +16,25 @@ public class GoogleCredentialProvider
     {
         public static GoogleCredential FetchCredential(IConfiguration configuration)
         {
+            string foo = configuration["foo"];
```

Comment:
Um?

Reply:
Whoops. That's left over as a test that the indexer returns null if the key's not found. Will remove :)